bar-chart-race/src/BarChartRace.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import BarChart from 'chart-race-react';
+import data from './data';
+
+const randomColor = () => {
```

Comment:
I would avoid using random color, we could use color scheme like D3 does. You can copy D3 color schemes from this site:

https://observablehq.com/@d3/color-schemes

You could use categorical10 and only read until 10 category columns from the dataset